pkg/messagebroker/kafka.go

```diff
@@ -218,7 +218,7 @@ func (k *KafkaBroker) CreateTopic(ctx context.Context, request CreateTopicReques
 	ts := kafkapkg.TopicSpecification{
 		Topic:             tp,
 		NumPartitions:     request.NumPartitions,
-		ReplicationFactor: 1,
+		ReplicationFactor: len(k.Config.Brokers),
```

Comment:
high value will have impact on publish latency as data will need time to replicate in sync. we should start with atleast 50% number 

i.e. (len(k.Config.Brokers)+1)/2

Reply:
Plus more replication will add to cost as well in terms of storage and bytes over network